### DISCUSSION TITLE
请教怎么修改popover弹出框的圆角大小

### DISCUSSION THREAD

**Discussion by wolf92** [youzan/vant#10150] [Q&A]
使用v-deep方式，修改圆角大小，不生效呢？style是<style lang="scss" scoped>，还不能改
::v-deep .van-popover__content {
        border-radius:1px;
}

**Comment by ydc544696000** [comment]
只有写全局才能改 我发现popover组件和#app是同一级，所以用deep无效，但是有新的问题，如果内容过多，我限制高度让其出现滚动条之后，移动端滑动有时会无效

**Reply by liubaoheng** [reply]
那你是怎么解决的啊，我也是遇到这样的问题了

**Reply by ydc544696000** [reply]
如果你是指样式修改无效，你可以尝试不用scope属性，直接写全局样式（不用加deep），不过这样会有样式污染。
如果你是指修改样式后出现滚动条但是滚动偶尔失灵的问题，我当时的业务场景是在dialog中使用popover，而dialog默认背景锁定，阻止了滑动行为，所以可以打开dialog的背景滚动（lockScroll属性），这样背景就会跟随滚动，这时候你解决滚动穿透的问题就可以了（解决滚动穿透的方法可以搜索一下，方法比较多）
